stats.py

```diff
@@ -0,0 +1,48 @@
+class WeatherStats:
+    @staticmethod
```

Comment:
Why Static method?

Reply:
I used static method here as this method did not need to access or change the class specific data. It computes result solely based on the given weather data as parameter, finds the required values and returns them without any need to interact with any class data.